src/common/AstroReactTable/AstroReactTable.tsx

```diff
@@ -53,26 +55,31 @@ export const AstroReactTable = ({
       </header>
 
       <div className='Astro-react-table__body'>
-        {table.getRowModel().rows.map(({ id, getVisibleCells, original }: any) => (
-          <div
-            key={id}
-            onClick={() => handleRowClick(original)}
-            className={classNames('Astro-react-table__row', {
-              'Astro-react-table__selected': handleIsSelected(original),
-              'Astro-react-table__selectable': !!onRowClick,
-            })}
-          >
-            {getVisibleCells().map(({ id, column, getContext }: any) => (
-              <div
-                key={id}
-                className='Astro-react-table__cell'
-                style={column.columnDef.style}
-              >
-                {flexRender(column.columnDef.cell, getContext())}
-              </div>
-            ))}
-          </div>
-        ))}
+        {table
+          .getRowModel()
+          .rows.map(({ id, getVisibleCells, original }: any) => (
+            <div
+              key={id}
+              onClick={() => handleRowClick(original)}
+              className={
+                handleIsSelected(original)
+                  ? 'Astro-react-table__row Astro-react-table__selected'
+                  : 'Astro-react-table__row' || !!onRowClick
+                  ? 'Astro-react-table__row Astro-react-table__selectable'
+                  : 'Astro-react-table__row'
+              }
```

Comment:
Same here, a template literal will simplify this a bit:

```suggestion
              className={`Astro-react-table__row ${
                handleIsSelected(original)
                  ? 'Astro-react-table__selected'
                  : !!onRowClick
                  ? 'Astro-react-table__selectable'
                  : ''
              }`}
```